manifests/cluster-infra/grafana_deployment.yaml

```diff
@@ -37,6 +37,8 @@ spec:
             value: "Main Org."
           - name: GF_SECURITY_ADMIN_PASSWORD
             value: "{{ .GRAFANA_ADMIN_PASSWORD }}"
+          - name: GF_USERS_VIEWERS_CAN_EDIT
```

Comment:
this means theat anon access also allows to edit the dashboards right?

I am not sure this is a good idea.

Reply:
They will be able to `view` and `edit` but not `save` , this option is required if we want to expose the grafana explorer page. (which we need to display loki logs publicly)

https://grafana.com/docs/installation/configuration/#viewers-can-edit

Reply:
No problem than.